### PR TITLE
fix task #30

### PR DESCRIPTION
**Pull request by blackout314**
- added link to table for every node task #30

### CODE REVIEW

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this pull request

**Comment by 0crat** [comment]
Job [#33](https://github.com/zold-io/zold.github.io/issues/33) is now in scope, role is `REV`

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) everybody who has role `REV` is banned at [#33](https://github.com/zold-io/zold.github.io/issues/33); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by yegor256** [comment]
@blackout314 why blicking code is also here?

**Comment by blackout314** [comment]
@yegor256 no conflict now

File: js/map.js [hunk 1]
```diff
@@ -78,17 +77,23 @@ function put_markers(map, remotes) {
...
 
     } else {
-      $('#remotes-table').append('<tr data-coords="' + coords + '"><td>' + coords + '</td> <td colspan=3>&nbsp;</td> </tr>')
+      $('#remotes-table').append('<tr data-coords="' + coords + '"><td>' + makeALink(coords) + '</td> <td colspan=3>&nbsp;</td> </tr>')
     }
   });
 }
 
+function makeALink(coords) {
+  return '<a target="_blank" href="http://' + coords + '">' + coords + '</a>';
```
Comment (yegor256) [inline js/map.js]: @blackout314 it's a bad practice to use `target` and it's not compliant with HTML5

**Comment by yegor256** [comment]
@blackout314 see abo ve

**Comment by blackout314** [comment]
@yegor256 ok

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@blackout314 we seem to have merge conflicts now

**Comment by blackout314** [comment]
@yegor256 resolved

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:zold-io/zold.github.io#33` is not assigned, can't get performer

**Comment by 0crat** [comment]
The job [#33](https://github.com/zold-io/zold.github.io/issues/33) is now out of scope

**Pull request merged**